src/main/java/appeng/parts/automation/StorageLevelEmitterPart.java

```diff
@@ -97,7 +99,11 @@ public void onStackChange(AEKey what, long amount) {
                 lastReportedValue = amount;
                 updateState();
             } else { // either fuzzy upgrade or null filter
-                updateReportingValue(getGridNode().getGrid());
+                long currentTick = TickHandler.instance().getCurrentTick();
```

Comment:
```suggestion
                // When using a fuzzy upgrade or no filter at all, the level emititer will actively scan the grid
                // We need to ensure we only do this once per tick in case any stack has changed.
                long currentTick = TickHandler.instance().getCurrentTick();
```